e2e/LongHaul/prerequisites/LongHaulSetup.ps1

```diff
@@ -232,7 +222,7 @@ $longhaulDeviceModuleId = "LongHaulDeviceModule1"
 $longhaulEdgeDeviceId = "LongHaulEdgeDevice1"
 $longhaulEdgeModuleId = "LongHaulEdgeModule1"
 $longhaulDevice = az iot hub device-identity list -g $ResourceGroup --hub-name $iotHubName --query "[?deviceId=='$longhaulDeviceId'].deviceId" --output tsv
-$longhaulDeviceModule = az iot hub module-identity list -g $ResourceGroup --hub-name $iotHubName --query "[?moduleId=='$longhaulDeviceModuleId'].moduleId" --output tsv
+$longhaulDeviceModule = az iot hub module-identity list -g $ResourceGroup --hub-name $iotHubName --device-id $longhaulDeviceId --query "[?moduleId=='$longhaulDeviceModuleId'].moduleId" --output tsv
```

Comment:
"module-identity list" requires the associated deviceID. Similar to the "longhaulEdgeModule" script below.